assets/js/googlesitekit/widgets/components/URLSearchWidget.js

```diff
@@ -49,7 +49,7 @@ function URLSearchWidget() {
 	return (
 		<div className="mdc-layout-grid__cell">
 			<Widget
-				slug="urlSearchWidget"
+				widgetSlug="urlSearchWidget"
```

Comment:
Here we should also be able to use `props.Widget` instead of the imported component (this way we also don't need to pass the slug anymore).